src/TConsole.cpp

```diff
@@ -2695,7 +2695,7 @@ void TConsole::slot_searchBufferUp()
             return;
         }
     }
-    print(tr("No search results, sorry!\n"));
+    print(QStringLiteral("No search results, sorry!\n"));
```

Comment:
This doesn't seem correct, because you didn't take the `\n` out of `tr()` but took out `tr()` entirely. Is this in error?

Reply:
I think too, that this is an error - IMO it should be:
```CPP
     print(QStringLiteral("%1\n").arg(tr("No search results, sorry!")));
```

:bulb: :thought_balloon: Since identical `QStringLiterals(`...`)` are stored separately within the read-only section of the executable code it is non-ideal to duplicate duplicated strings. Given that the "*append a new-line*" modification to strings is not an uncommon one we might consider defining a global (well semi-global):
```CPP
    (static const QString) appendLineFeed(const QString& text) = QStringLiteral("%1\n").arg(text);`
```
inside, say the `mudlet` class header file. So that this line (and the other one further down) could become:
```CPP
    print(mudlet::appendLineFeed(tr("No search results, sorry!")));
```

Reply:
Please keep in mind that this is a PR from someone who's quite new to this! Let's keep it simple for the time being and not go for the perfect solution.